content/en/security/application_security/enabling/tracing_libraries/code_security/_index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Enabling ASM Code Security using Datadog Tracing Libraries
+kind: documentation
+type: multi-code-lang
+further_reading:
+- link: "/security/application_security/"
+  tag: "Documentation"
+  text: "Protect against Threats with Datadog Application Security Management"
+- link: "/security/application_security/enabling/compatibility/"
+  tag: "Documentation"
+  text: "Programming Language and Framework Compatibility"
+- link: "/security/application_security/add-user-info/"
+  tag: "Documentation"
+  text: "Tracking user activity"
+- link: "/security/default_rules/#cat-application-security"
```

Comment:
```suggestion
- link: "/security/default_rules/?category=cat-application-security"
```
^ This is in a few of your further reading sections, but I think the URL is incorrect.